shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
actually, this will introduce problems: shell is dependent by integration tests. and we should not have ufs modules in integration tests.